pre_commit_dbt/utils.py

```diff
@@ -115,6 +124,13 @@ def paths_to_dbt_models(
     return [prefix + Path(path).stem + postfix for path in paths]
 
 
+def get_exposure_path(
```

Comment:
If my understanding is correct, you return a list here. So the type annotations may not be correct. And I think it could be a good idea to return a generator instead.

Reply:
Because generators are memory-efficient. On a large project, we may have a lot of exposures and so a lot of paths. With generators, there is roughly one path at the time in RAM when we iterate it. So it is generally a good practice to use generators instead of lists.

And if you absolutely need a list, you can use `list(my_generator)`.